packages/nim-serving/package.json

```
@@ -16,8 +16,10 @@
     "./types": "./src/api/nimservices/types.ts"
   },
   "dependencies": {
+    "@odh-dashboard/hardware-profiles-shared": "*",
     "@odh-dashboard/internal": "*",
     "@odh-dashboard/k8s-core": "*",
+    "@odh-dashboard/ui-core": "*",
```

Biome (2.5.1)

[error] The key `@odh-dashboard/ui-core` was already declared.

(lint/suspicious/noDuplicateObjectKeys)